src/VirtoCommerce.CatalogModule.Core/Model/OutlinePart/BaseOutlinePartResolver.cs

```diff
@@ -0,0 +1,22 @@
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.CatalogModule.Core.Model.OutlinePart
```

Comment:
In the case of creating a base class probably, you could inherit from `IOutlinePartResolver`, implementing  `ResolveOutlineName` method and leaving `ResolveOutlinePart` abstract.
And probably it should be called `BaseNameOutlinePartResolver`